pages/common/pio-ci.md

```diff
@@ -0,0 +1,29 @@
+# pio ci
+
+> Build PlatformIO projects with arbitrary source code structure.
+> This will create a new temporary project, which the source code will be copied into.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_ci.html>.
+
+- Build a PlatformIO project in the temporary directory of the operating system and delete it afterwards:
+
+`pio ci {{path/to/project}}`
+
+- Build a PlatformIO project and specify specific libraries:
+
+`pio ci --lib {{path/to/library_directory}} {{path/to/project}}`
```

Comment:
I was wondering if more than one lib is supported .. something like:
```suggestion
`pio ci --lib {{path/to/library_directory1}} --lib {{path/to/library_directory2}} {{path/to/project}}`
```

Reply:
You can specify almost everything multiple times, but I think that makes it way harder to read. If someone is interested in the details, we have the link to the documentation in the description.

Reply:
I agree .. You are correct :) .. let's not make it too verbose